### PR TITLE
Feature update css nuk

### PR DESCRIPTION
**Pull request by codenuk**

### CODE REVIEW

File: src/components/admin-login/index.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ const AdminLoginComponent = () => {
             <form className={styles.formAdminLogin}>
                 <h3>Stickerwish Admin Login</h3>
                 <label>อีเมล</label>
-                <TextInput name="email" />
+                <TextInput className="inputText" name="email" />
```
Comment (Knight-H) [inline src/components/admin-login/index.js]: Not field?

File: src/components/approve-layout/index.js [hunk 2]
```diff
@@ -30,142 +24,25 @@ const ApproveLayoutComponent = () => {
             }
 
             <h1 className={styles.title}>รายการออเดอร์</h1>
-            <p>ออเดอร์หมายเลข #DW0001
-                <label className={styles.waitApproval}>กำลังดำเนินการ</label>
+            <p>ออเดอร์หมายเลข {fakeAPI[0].orderNumber}
+                {/* <label className={styles.waitApproval}>กำลังดำเนินการ</label> */}
+                <LabelSatus status={fakeAPI[0].statusOrder}/>
```
Comment (Knight-H) [inline src/components/approve-layout/index.js:29]: LABELSATUS?

File: src/components/common/formik-password-input.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const PasswordInput = ({ ...props }) => {
 
     return (
         <>
-            <input type='password' className="inputText" {...field} {...props} />
+            <input type='password' {...field} {...props} />
```
Comment (Knight-H) [inline src/components/common/formik-password-input.js]: just use field

File: src/components/common/formik-radio-input.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useField } from 'formik';
```
Comment (Knight-H) [inline src/components/common/formik-radio-input.js]: use `<Field/>`

File: src/components/navbar/index.js [hunk 5]
```diff
@@ -66,7 +66,7 @@ const NavBarComponent = ({ history }) => {
                     <li><Link to={{pathname: "/", 
                     // hash: "#ourWorks",  
                     state: {scrollToOurWorks: true}}}>ตัวอย่างผลงาน</Link></li>
-                    <li><button><Link to="/order-1-product-config">สั่งทำสติกเกอร์</Link></button></li>
+                    <li><button><Link to="/form_step_shopping">สั่งทำสติกเกอร์</Link></button></li>
```
Comment (SorawitSiri) [inline src/components/navbar/index.js]: Link ต้องทำใช้ history ของ react หรือป่าว

File: src/components/group-delivery-payment/index.module.scss [hunk 6]
```diff
@@ -0,0 +1,119 @@
+.groupDelivery {
+    flex-basis: 50%;
+
+    .containInformationCustomer{
```
Comment (Knight-H) [inline src/components/group-delivery-payment/index.module.scss]: container

File: src/components/group-delivery-payment/index.module.scss [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+            padding: 0;
+            list-style-type: none;
+            li {
+                font-size: calc(10px + 0.3vw);
+                margin: 0 0 15px 0;
+            }
+        }
+    }
+
+    .containInformation {
```
Comment (Knight-H) [inline src/components/group-delivery-payment/index.module.scss]: container

File: src/components/helper.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
+export const validateFiledID = (value) => {
```
Comment (Knight-H) [inline src/components/helper.js]: file

File: src/components/navbar/index.js (hunk 5 above)
Comment (SorawitSiri) [reply src/components/navbar/index.js]: ใช้ button onClick=history.push(????)

File: src/components/helper.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+        if (obj.hasOwnProperty(key))
+            return false;
+    }
+    return true;
+}
+
+export const isObject = (obj) =>
+    obj !== null && typeof obj === 'object';
+    
+export function setNestedObjectValues(
```
Comment (Knight-H) [inline src/components/helper.js]: now no need

File: src/components/home-member/index.module.scss [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+
+    svg {
+        margin-right: 5px;
+        width: 20px;
+        height: 20px;
+    }
+}
+
+.contain {
```
Comment (Knight-H) [inline src/components/home-member/index.module.scss]: container

File: src/components/home-member/index.module.scss [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+  }
+  .blueStatus {
+    background-color: #6DB9FF;
+  }
+  .yellowStatus {
+    background-color: #FFE146;
+  }
+  .orangeStatus {
+    background-color: #FF9F46;
+  }
```
Comment (Knight-H) [inline src/components/home-member/index.module.scss:143]: formatting

File: src/components/home-member/index.module.scss [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+  
+@media only screen and (max-width: 320px) {
+    .contain {
```
Comment (Knight-H) [inline src/components/home-member/index.module.scss]: container

File: src/components/home/index.js [hunk 13]
```diff
@@ -94,7 +94,7 @@ const HomeComponent = (props) => {
                     </ol>
                     <button>
                         <S21OrderIcon />
-                        <Link to="/order-1-product-config"><b>สั่งซื้อสินค้า</b></Link>
+                        <Link to="/form_step_shopping"><b>สั่งซื้อสินค้า</b></Link>
```
Comment (Knight-H) [inline src/components/home/index.js]: ???

File: src/components/navbar/index.js (hunk 5 above)
Comment (Knight-H) [inline src/components/navbar/index.js]: customize

File: src/components/order-1-product-config/index.js [hunk 14]
```diff
@@ -1,83 +1,107 @@
...
+                            <TextInput name="widthStricker" validate={validateWidthFiledID} />
+                            <TextInput name="heightStricker" validate={validateHeightFiledID} />
                         </div>
-                        
                     </div>
 
                     <div className={styles.dropdownSelect}>
                         <label htmlFor="quantity">จำนวน</label>
-                        <select id="quantity">
+                        <SelectInput name="quantityStricker" validate={validateQuantityFiledID}>
```
Comment (Knight-H) [inline src/components/order-1-product-config/index.js]: Field

**Review by SorawitSiri — COMMENTED** [review]
Must edit button for Link.

File: src/components/order-1-product-config/index.js [hunk 15]
```diff
@@ -1,83 +1,107 @@
...
+                            (err) => {
+                                setTouched(setNestedObjectValues(values, true))
+                                setErrors(err);
+                                if (isEmpty(err)) {
+                                    setFieldValue("stepProgress", 1, false)
+                                }
+                            }
+                        )}>
                         ถัดไป
                     </button>
```
Comment (Knight-H) [inline src/components/order-1-product-config/index.js]: type="submit"

File: src/components/order-1-product-config/index.js [hunk 16]
```diff
@@ -1,83 +1,107 @@
...
                         </button>
                     </div>
-                    
-                    <Link className={styles.link} to="/upload-file">
-                    <button className={styles.nextButton}>
+
+                    <button type="button" className={styles.nextButton}
+                        onClick={() => validateForm().then(
+                            (err) => {
+                                setTouched(setNestedObjectValues(values, true))
```
Comment (Knight-H) [inline src/components/order-1-product-config/index.js]: NO NEED

File: src/components/preview-image/index.js [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+        <>
+            <div className={styles.square}>
+                <Drawing />
+            </div>
+
+            <div className={styles.inputBox}>
+                <button><h3><IconCheckSVG /> อนุมัติแบบ</h3></button>
+                <input className={styles.inputGreen} type="text" placeholder="พิมพ์ข้อความ..."></input>
+
+                <div className={styles.groupBtn}>
```
Comment (Knight-H) [inline src/components/preview-image/index.js]: Btn or Button

File: src/components/preview-image/index.js [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+                    <button type="button">ส่ง</button>
+                    <button type="button" className={styles.btnCustomWidth}>อัพโหลดไฟล์</button>
```
Comment (Knight-H) [inline src/components/preview-image/index.js]: btn in front or at the back

File: src/components/upload-file/index.js [hunk 19]
```diff
@@ -67,19 +66,19 @@ const UploadFileComponent = () => {
...
-                        <button className={styles.btnCart}>
-                            <IconCart /><b>ใส่ในตะกร้า</b>
-                        </button>
-                    </div>*/}
+                    <button type="submit" className={styles.btnCart}
+                        onClick={() => {
+                            if (!values.uploadFileStricker && !values.approvalStricker) {
+                                setFieldValue("stepProgress", 2, false)
+                            } else {
+
```
Comment (Knight-H) [inline src/components/upload-file/index.js]: cahnge to API

**Review by Knight-H — CHANGES_REQUESTED** [review]

**Review by SorawitSiri — DISMISSED** [review]

**Review by SorawitSiri — APPROVED** [review]

**Review by Knight-H — APPROVED** [review]

**Pull request merged**